cmd/mimir/help.txt.tmpl

```diff
@@ -87,6 +87,8 @@ Usage of ./cmd/mimir/mimir:
     	Comma-separated list of network CIDRs to block in Alertmanager receiver integrations.
   -alertmanager.receivers-firewall-block-private-addresses
     	True to block private and local addresses in Alertmanager receiver integrations. It blocks private addresses defined by  RFC 1918 (IPv4 addresses) and RFC 4193 (IPv6 addresses), as well as loopback, local unicast and local multicast addresses.
+  -alertmanager.sharding-ring.consul.acl-token string
```

Comment:
It's categorized as advanced. It shouldn't be here.

Reply:
We need to distinguish between outer struct and inner field. Right now categorization finds the inner field with same address as outer struct, and replaces the field information. If we make inner field unexported, we can ignore such field in Mimir's code generating flag. I've posted PR https://github.com/grafana/dskit/pull/154 to accomplish that.

Reply:
How cool is the fact that we have a preview of the `-help` in `cmd/mimir/help.txt.tmpl`? :D

Reply:
😄